cmd/provenanced/cmd/genaccounts.go

```diff
@@ -420,7 +420,9 @@ enforced immediately.  An optional type flag can be provided or the default of C
 				managerAddr,
 				accessGrants,
 				markerStatus,
-				markertypes.MarkerType(markerType))
+				markertypes.MarkerType(markerType),
+				true,
```

Comment:
Is it always a fixed supply?

Reply:
For this it currently is.  If you look in the marker types, I changed a constructor `NewMarkerAccount` to take in a fixed supply parameter that was automatically set to true before.  I didn't want to change the behavior for anything that was out of scope for the ticket with this minor refactor.  So, I just hard coded the true here.  However, if you look in the msg_server.  We have a few instances of using `NewMarkerAccount` then the next line would change the supply fixed value.  Now that is fixed to just take it in as a param.